Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -639,6 +637,11 @@ public async Task Init(IServiceCollection serviceCollection)
                 {
                     services.AddResponseCompression();
                     services.AddHttpContextAccessor();
+                    services.AddMvc()
+                        .SetCompatibilityVersion(CompatibilityVersion.Version_2_2)
+                        .AddApplicationPart(Assembly.Load("Jellyfin.Api")).AddControllersAsServices(); // TODO netcoreapp doesn't need this
```

Comment:
can you explain what this `TODO` means?

Reply:
In netcoreapp it should automatically register apicontroller from other projects/assemblies from what I've read. It's not ideal having it hardcoded to a specific assembly